src/main/java/wellnus/reflection/command/UnlikeCommand.java

```diff
@@ -0,0 +1,185 @@
+package wellnus.reflection.command;
+
+import java.util.HashMap;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import wellnus.command.Command;
+import wellnus.exception.BadCommandException;
+import wellnus.exception.StorageException;
+import wellnus.exception.TokenizerException;
+import wellnus.reflection.feature.IndexMapper;
+import wellnus.reflection.feature.QuestionList;
+import wellnus.reflection.feature.ReflectUi;
+
+//@@author wenxin-c
+/**
+ * Unlike command to remove reflection questions from favorite questions.
+ */
+public class UnlikeCommand extends Command {
+    private static final String COMMAND_DESCRIPTION = "unlike (index) - Remove a particular question "
+            + "from favorite list.";
+    private static final String COMMAND_USAGE = "usage: unlike (index)";
+    private static final String COMMAND_KEYWORD = "unlike";
+    private static final String FEATURE_NAME = "reflect";
+    private static final String INVALID_COMMAND_MSG = "Command is invalid.";
+    private static final String INVALID_COMMAND_NOTES = "Please check the available commands "
+            + "and the format of commands.";
+    private static final String WRONG_INDEX_MSG = "Please input the correct index of the question you "
+            + "remove from favorite list!";
```

Comment:
Seems to have a grammatical error here: "question you **want to** remove from **your** favorite list!"